Israel National Newspaper Library.js

```diff
@@ -0,0 +1,143 @@
+{
+	"translatorID": "9d8099b7-1c50-4159-9a67-f9fc1fb3b463",
+	"label": "Israel National Newspaper Library",
+	"creator": "Anonymus",
+	"target": "^https:\\/\\/www\\.nli\\.org\\.il\\/(en|he|ar)\\/newspapers\\/.*",
```

Comment:
Thanks for changing this one. I think it's still too specific, so I'd suggest the following change + a corresponding on in the `detectWeb()` (see other comments). That will make the translator work for the whole library and make it simpler to extend it in the future (no need to edit the `target` + `detectWeb()` each time, everything will be in the latter).


```suggestion
	"target": "^https:\\/\\/www\\.nli\\.org\\.il\\/",
```